src/schema/sources.ts

```diff
@@ -926,13 +933,23 @@ export const resolvers: IResolvers<any, Context> = {
             .andWhere(`${builder.alias}."sourceId" = :source`, {
               source: sourceId,
             })
-
             .addOrderBy(
               graphorm.mappings.SourceMember.fields.roleRank.select as string,
               'DESC',
             )
             .addOrderBy(`${builder.alias}."createdAt"`, 'DESC');
 
+          if (query) {
+            builder.queryBuilder = builder.queryBuilder
+              .innerJoin(User, 'u', `${builder.alias}."userId" = u.id`)
+              .andWhere(
+                `(REPLACE(u.name, ' ', '') ILIKE :name OR  u.username ILIKE :name)`,
```

Comment:
My concern is that it is not indexed properly but that's fine for now I guess.
Anyway, found a double space:

```suggestion
                `(REPLACE(u.name, ' ', '') ILIKE :name OR u.username ILIKE :name)`,
```

Reply:
Yep, I agree. I think it should be fine for now and the pool of users won't be too large as most members don't exceed 1k, yet.